appserver/extras/docker-images/server-node/src/main/docker/Dockerfile

```diff
@@ -14,20 +11,10 @@ ENV PAYARA_PASSWORD_FILE=${PAYARA_PASSWORD_FILE_DIR}/passwordfile.txt
 
 # Create and set the Payara user and working directory owned by the new user
 RUN true \
-    && mkdir ${HOME_DIR} \
-    && mkdir ${PAYARA_DIR} \
-    && mkdir ${PAYARA_PASSWORD_FILE_DIR} \
-    && groupadd -g 1000 payara \
-    && useradd -u 1000 -M -s /bin/bash -d ${HOME_DIR} payara -g payara \
-    && echo payara:payara | chpasswd \
-    && chown -R payara:payara ${HOME_DIR} \
+    && mkdir -p ${PAYARA_PASSWORD_FILE_DIR} \
     && true
 
-USER payara
-WORKDIR ${HOME_DIR}
+COPY --chown=payara:payara maven/artifacts/@docker.payara.rootDirectoryName@ ${PAYARA_DIR}
```

Comment:
Revert order of these 2 COPY commands as script content will change less likely as Payara Micro. -> reuse of Server